packages/client/src/createClient.ts

```diff
@@ -186,16 +186,20 @@ export function createClientInternal(
               const P extends PropertyKeys<Q>,
             >(
               objectSet: ObjectSet<Q>,
-              properties: Array<P>,
-              listener: EXPERIMENTAL_ObjectSetListener<Q, P>,
+              listener: ObjectSetListener<Q, P>,
+              opts?: {
+                properties?: Array<P>;
+                requestLatestGeotimeSeriesValues?: boolean;
```

Comment:
I made this ultra specific for right now. I think this is the best option for right now since this is the only property that we can support and it makes it clear what they're going to get back for right now. 

My main concern is that this will cause a break when more of property types are supported here. There are two things I have in mind for that 1. We just rename this to the greatest scope we have or 2. We add another opt specifically for that property type. I'm in favor of 1 I'm just not sure what the next addition will be

Reply:
From reading the name it is not clear to me what this does. I had to read the code to understand it. 

I think we need to enumerate out what people want to do. 

For `$(Geolocatable).subscribe(myListener)`, should the references be retrieved automatically? What is my likely intent?

What about `$(Geolocatable).subscribe(myListener, { properties: ["location"] })`? Is the answer different this time if `location` is or isnt a geotime?



I'm intentionally not answering these questions myself because I want you to work through the options.